modules/store/spec/utils.spec.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @fileOverview
+ * @author Brian Frichette (brian@eturi.com)
+ */
+
+import { omit } from '../src/utils';
+import { combineReducers, compose } from '@ngrx/store';
+
+describe(`Store utils`, () => {
+  describe(`combineReducers()`, () => {
+    const s1 = { x: '' };
+    const s2 = { y: '' };
+    const r1 = (s = s1, a: any): typeof s1 =>
```

Comment:
Just a preference, but make the variable names more descriptive than `s` and `a` so they are more readable at glance